package.json

```diff
@@ -39,24 +42,32 @@
     "hapi": "^8.4.0",
     "hoek": "^2.11.1",
     "hyperquest": "^1.0.1",
-    "imports-loader": "^0.6.3",
     "joi": "^6.0.8",
-    "json-loader": "^0.5.1",
-    "jsx-loader": "^0.12.2",
-    "less": "^2.4.0",
-    "parallelshell": "^1.1.1",
-    "react": "^0.13.1",
-    "react-router": "^0.13.1",
-    "react-validation-mixin": "^4.0.3",
-    "webpack": "^1.7.3"
+    "react": "^0.12.2",
+    "react-router": "^0.12.4",
+    "react-validation-mixin": "^4.0.3"
   },
   "devDependencies": {
     "autoless": "^0.1.6",
+    "bulk-require": "^0.2.1",
     "code": "^1.3.0",
+    "imports-loader": "^0.6.3",
+    "js-beautify": "^1.5.5",
+    "jscs": "^1.11.3",
+    "jshint": "^2.6.3",
+    "json-loader": "^0.5.1",
+    "jsx-loader": "^0.12.2",
     "lab": "^5.5.0",
+    "less": "^2.4.0",
```

Comment:
We don't commit `styles.css` in the repo and I'm just concerning about building the app to heroku where it will fail when we are trying to do `npm start` which suppose to build `less` and `build webpack`